package.json

```diff
@@ -65,7 +65,7 @@
   },
   "scripts": {
     "build": "npm-run-all clean copy build:app",
-    "build:app": "obt build --js=./client/src/main.js --sass=./client/src/main.scss --buildFolder=./client/dist && cp ./client/src/bookmarklet.js ./client/dist && cp -r ./client/src/images ./client/dist && gulp set-service-url && gulp one-line-bookmark",
+    "build:app": "mkdir ./client/dist && cp ./client/src/bookmarklet.js ./client/dist && cp -r ./client/src/images ./client/dist && gulp set-service-url && gulp one-line-bookmark",
```

Comment:
It didn't create the directory on my system, the output was dumped into a file instead. However, it may just be that the copy command (which I didn't notice) needs a './' in the path to behave.